<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableExportWidget</class>
 <widget class="QWidget" name="TableExportWidget">
  <layout class="QGridLayout" name="export_layout">
   <item row="0" column="0">
    <widget class="QLabel" name="export_delimiter_label">
     <property name="text">
      <string>Delimiter:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="export_delimiter_combo">
     <item>
      <property name="text">
       <string>Comma</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tab</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="export_line_ending_label">
     <property name="text">
      <string>Line Ending:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="export_line_ending_combo">
     <item>
      <property name="text">
       <string>LF (\n)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CRLF (\r\n)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="export_precision_label">
     <property name="text">
      <string>Decimal Precision:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="export_precision_spinbox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="export_header_checkbox">
     <property name="text">
      <string>Include Header Row</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="export_by_group_checkbox">
     <property name="text">
      <string>Export Separate Files by Entity Group</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>When enabled, creates separate CSV files for each entity group instead of a single file</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="export_csv_btn">
     <property name="text">
      <string>Export to CSV...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>


